[EWS][WPE] WPE API testers should use custom command like GTK ones

clopez opened the pull request:
#### a2e919721fc064de008d051a4e09b8d24d8070df
<pre>
[EWS][WPE] WPE API testers should use custom command like GTK ones
<a href="https://bugs.webkit.org/show_bug.cgi?id=263086">https://bugs.webkit.org/show_bug.cgi?id=263086</a>

Reviewed by Aakash Jain.

On 269162@main new WPE API testers were added to the EWS.
And when testing it seems things are failing because the new
workers are trying to run the API tests with the default script
Tools/Scripts/run-api-tests (used by Mac workers) instead of with
the script Tools/Scripts/run-${port}-tests used by GTK and WPE
workers.

* Tools/CISupport/ews-build/steps.py:
(RunAPITests.run):
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: <a href="https://commits.webkit.org/269274@main">https://commits.webkit.org/269274@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/9ea908f75c196c35d0eba296115e873c508893bb

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/22087 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/22297 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/23156 "Built successfully") | [❌ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/23972 "Failed to checkout and rebase branch from PR 19014") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/20446 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/22332 "Passed tests") | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/23/builds/26547 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/16/builds/22615 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/5/builds/23972 "Failed to checkout and rebase branch from PR 19014") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/22317 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/23/builds/26547 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/19165 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/2/builds/24825 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/23/builds/26547 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/20019 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/2/builds/24825 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/23/builds/26547 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/20244 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/2/builds/24825 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/20674 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/16/builds/22615 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/21892 "Passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/20027 "Built successfully") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/4/builds/24235 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/2749 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/20622 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aj062 reviewed (approved):
rs=me

aj062 commented on Tools/CISupport/ews-build/steps.py:
```diff
@@ -5041,8 +5041,8 @@ def run(self):
-        if platform == 'gtk':
-            self.command = ['python3', 'Tools/Scripts/run-gtk-tests',
+        if platform in ['gtk', 'wpe']:
+            self.command = ['python3', 'Tools/Scripts/run-{0}-tests'.format(platform),
```
while you are at it, can also use f-string here.

clopez replied:
sure! thanks for the tip 😀

clopez committed:
[EWS][WPE] WPE API testers should use custom command like GTK ones
https://bugs.webkit.org/show_bug.cgi?id=263086

Reviewed by Aakash Jain.

On 269162@main new WPE API testers were added to the EWS.
And when testing it seems things are failing because the new
workers are trying to run the API tests with the default script
Tools/Scripts/run-api-tests (used by Mac workers) instead of with
the script Tools/Scripts/run-${port}-tests used by GTK and WPE
workers.

* Tools/CISupport/ews-build/steps.py:
(RunAPITests.run):
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: https://commits.webkit.org/269274@main
